lib/hanami/assets/helpers.rb

```diff
@@ -682,6 +752,15 @@ def _typed_asset_path(source, ext)
         asset_path(source)
       end
 
+      def _sri?
+        !!self.class.assets_configuration.sri
+      end
+
+      def _sri_value(source, ext)
+        source = "#{ source }#{ ext }" unless source.match(/#{ Regexp.escape(ext) }\z/)
+        self.class.assets_configuration.sri_value(source) unless _absolute_url?(source)
```

Comment:
Yes it is, although in that case an explicit SRI value can be included via the helper:

```ruby
<%= javascript 'https://code.jquery.com/jquery-2.2.0.min.js', sri: 'sha512-hash-provided-by-jquery-developers' %>
```